src/components/Hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from "react";
+
+const useLocalStorage = <T>(key: string, initialValue: T): [T, (newValue: T) => void] => {
+    const [value, setValue] = useState<T>(initialValue);
```

Comment:
is `<T>` not inferred? I don't know how good TS is at this sort of thing, but assume not - just wanted to check!

Reply:
Yep it is